rfcs/2021-11-whcm-support.md

```diff
@@ -0,0 +1,185 @@
+
+For example, if we wanted to increase the border-width on `<FilterChip />`, you could write the following code:
+
+```css
+@media (-ms-high-contrast: active), (forced-colors: active) {
+  .ds-c-filter-chip__button {
+    border-width: 2px;
+  }
+}
+```
+
+In this example, a 2px border would be applied to the `<FilterChip />`, but only when WHCM is enabled.
+
+> ⚠️ It's worth noting that while you can apply specific color values (i.e., hex values) using this query, it's recommended you avoid using static colors in forced colors mode due to the risk of introducing a visual contrast bug.
+>
+> To implement colors, it's recommended you use CSS System Colors, which will automatically pick up whichever color the user specified. See Ex. 2 for more details about CSS System Colors.
+
+#### Ex. 2: Overriding an element's style rules using [CSS System Colors](https://www.w3.org/TR/css-color-4/#css-system-colors)
+
+For instances where the color of an element needs to be adjusted, it's recommended you use CSS System Colors. These colors are basically a limited set of variables a user sets to define their contrast theme. You can find a list of available colors in the link above.
+
+Using system colors in lieu of static colors is preferred because a static color, like `#ff0000`, may have unexpected and severe contrast issues for your user.
+
+For example, because we've implemented custom checkbox and radio buttons, these elements don't receive all the styles a native checkbox or radio button would normally receive. So when a checkbox or radio button is disabled, that style isn't apparent at all in WHCM. To fix something like this, you'd implement the following:
+
+```css
+@media (-ms-high-contrast: active), (forced-colors: active) {
+  .ds-c-choice:disabled + label {
+    color: GrayText; /* `GrayText` is the CSS System Colors name for disabled text */
+  }
+  .ds-c-choice:disabled + label:before {
+    border-color: GrayText;
+  }
+}
+```
+
+#### Ex. 3: Making sure an element's style persists, regardless of contrast mode
+
+There are some scenarios where you may want to force an element to appear the same in WHCM as you would in any non-WHCM environment. For our purposes, we may want to use this approach for sections of our documentation site, like our [color swatches](https://design.cms.gov/styles/color/), which have visual meaning but don't appear in WHCM due to the limited WHCM color palette.
+
+To preserve the original appearance of an element in WHCM, you could write the following:
+
+```css
+@media (-ms-high-contrast: active), (forced-colors: active) {
+  .c-swatch__preview {
+    -ms-high-contrast-adjust: none; /* For IE */
+    forced-color-adjust: none; /* For everything else * /
+  }
+}
+```
+
+### Recommendation
+
+**WHCM updates should support IE in addition to modern browsers.**
```

Comment:
I think these should be headings like your examples, and it would also be helpful to number them in my opinion